docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -0,0 +1,140 @@
+
+1. Initialize Widget Viewer using `widget_viewer_evas_init()`. The parameter of `widget_viewer_evas_init()` is the window object of an application:
+
+   ```
+   Evas_Object *win = elm_win_add(NULL, "VIEWER", ELM_WIN_BASIC);
+   int ret = widget_viewer_evas_init(win);
+   ```
+
+2. Create a widget object using `widget_viewer_evas_add_widget()`. The parameters of `widget_viewer_evas_add_widget()` are parent Evas_Object, widget ID, content information, and widget update period. The widget ID is the widget application's app ID or class ID. Content information is the bundle encoded data, which contains extra information on the widget. You can retrieve the content information using `widget_viewer_evas_get_content_info()`:
+
+
+   ```
+   const char *content_info;
+   bundle_raw *raw;
+   int len;
+
+   bundle_encode(data, &raw, &len);
+   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
+   evas_object_resize(widget, 100, 100);
+   evas_object_show(widget);
+   ```
+
+3. Finalize Widget Viewer functionality using `widget_viewer_evas_fini()`:
+
+   ```
+   int ret = widget_viewer_evas_fini();
+   ```
+
+<a name="manage_life-cycle"></a>
+## Manage widget life-cycle
+
+After displaying a widget, you can manage the widget application's lifecycle:
+
+1. Create widget object with `widget_viewer_evas_add_widget()`.
+
+   ```
+   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
+   ```
+
+2. Resume the instance of widget application using `widget_viewer_evas_resume_widget()` with the widget object as the parameter.:
+
+   ```
+   int ret = widget_viewer_evas_resume_widget(widget);
+   ```
+
+3. Pause the instance of widget application using `widget_viewer_evas_pause_widget()` with the widget object as the parameter.:
+
+   ```
+   int ret = widget_viewer_evas_pause_widget(widget);
+   ```
+
+4. Resume or Pause all the widget instances displayed by the widget viewer application using `widget_viewer_evas_notify_resumed_status_of_viewer()` and `widget_viewer_evas_notify_paused_status_of_viewer()`.:
+
+   ```
+   int ret = widget_viewer_evas_notify_resumed_status_of_viewer();
+   ret = widget_viewer_evas_notify_paused_status_of_viewer();
+   ```
+
+<a name="retrieve_information"></a>
+## Retrieve Widget Information
+
+After displaying widget, you can retrieve information about widget instances as shown in the following code.
+
+1. Create widget object with `widget_viewer_evas_add_widget()`.
+
+   ```
+   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
+   ```
+
+2. Get the content information using `widget_viewer_evas_get_content_info()` with the widget object as the parameter.:
```

Comment:
After parameter, a period (.) and a colon (:) is there.
Please remove (.) and keep (:) only.

Please consider this for 119 and 125 also.